packages/cardano-graphql-provider/src/cardanoGraphqlProvider.ts

```diff
@@ -3,6 +3,12 @@ import { gql, GraphQLClient } from 'graphql-request';
 import { TransactionSubmitResponse } from '@cardano-graphql/client-ts';
 import { Schema as Cardano } from '@cardano-ogmios/client';
 
+/**
```

Comment:
@sweeetland 

We should aim to provide a description and example for most functions or modules